astropy/visualization/wcsaxes/core.py

```diff
@@ -437,6 +437,18 @@ def grid(self, draw_grid=True, **kwargs):
         draw_grid : bool
             Whether to show the gridlines
         """
+        # these kwargs mock up the interface to plt.grid, allowing it to be used
+        # to set the grid
+        which = kwargs.pop('which', 'major')
+        if which != 'major':
+            raise ValueError('WCSAxes can only draw major gridlines')
+        axis = kwargs.pop('axis', 'both')
+        if axis != 'both':
+            raise ValueError('WCSAxes can only draw gridlines on both axes')
+        b = kwargs.pop('b', True)
+        if not b:
+            raise ValueError('WCSAxes cannot have grids removed once they are on')
```

Comment:
Also not quite true, normally setting ``draw_grid`` to ``False`` should in principle turn off the grid. But if it doesn't work, it's a bug :)